### PR TITLE
Chore states to exercise

### PR DESCRIPTION
**Pull request by fedescarpa**

### CODE REVIEW

File: lib/bibliotheca/schema/exercise.rb [hunk 1]
```diff
@@ -31,7 +31,9 @@ def self.fields_schema
 
       {name: :description, kind: :file, extension: 'md'},
       {name: :hint, kind: :file, extension: 'md'},
-      {name: :corollary, kind: :file, extension: 'md'}
+      {name: :corollary, kind: :file, extension: 'md'},
+      {name: :initial_state, kind: :file, extension: 'md'},
+      {name: :final_state, kind: :file, extension: 'md'}
```
Comment (fedescarpa) [inline lib/bibliotheca/schema/exercise.rb]: @flbulgarelli Initial and final board should be `md` or `yml`?

Comment (flbulgarelli) [reply lib/bibliotheca/schema/exercise.rb]: md

File: lib/bibliotheca/exercise.rb [hunk 2]
```diff
@@ -4,6 +4,7 @@ def initialize(e)
       @guide = e.indifferent_delete(:guide)
       super(e)
       process_choices!
+      set_boards!
```
Comment (flbulgarelli) [inline lib/bibliotheca/exercise.rb]: Set states

File: lib/bibliotheca/exercise.rb [hunk 3]
```diff
@@ -55,6 +56,27 @@ def markdownified
...
+    def to_gs_board(board)
+      "<gs-board> #{board} </gs-board>" if board
+    end
+
+    def boom_board
+      "<img src='https://user-images.githubusercontent.com/1631752/37945593-54b482c0-3157-11e8-9f32-bd25d7bf901b.png'>"
+    end
+
+    def kids?
+      effective_language_name == 'gobstones' && editor == 'custom'
```
Comment (flbulgarelli) [inline lib/bibliotheca/exercise.rb]: We should split this logic

**Comment by flbulgarelli** [comment]
Anyway, i Wil refactor it today.

**Comment by flbulgarelli** [comment]
And se should add a test

File: lib/bibliotheca/schema/exercise.rb (hunk 1 above)
Comment (fedescarpa) [reply lib/bibliotheca/schema/exercise.rb]: Perfect!

**Pull request merged**